app/services/twilio_service.rb

```diff
@@ -37,6 +37,12 @@ def new_offer_alert(offer)
     TwilioJob.perform_later(options)
   end
 
+  def send_unread_message_sms(url)
+    return unless allowed_to_send?
+    options = { to: @user.mobile, body: reminder_sms_for_unread_sms(url) }
```

Comment:
Suggest changing the name of the message body function.
```suggestion
    options = { to: @user.mobile, body: unread_message_reminder(url) }
```